services/repository/src/test/java/org/sagebionetworks/file/controller/UploadControllerTest.java

```diff
@@ -38,6 +40,8 @@
 	private S3FileHandle handleOne;
 	private PreviewFileHandle handleTwo;
 	private List<String> toDelete;
+	
+	private static final String BUCKET_NAME = StackConfigurationSingleton.singleton().getS3Bucket();
 

```

Comment:
is this change still needed?

Reply:
No, and this applies to three other tests which I've reverted as well.